Project 4/SPARQL-JV.md

```diff
@@ -0,0 +1,526 @@
+**The SPARQL Library of Buffalo**
+
```

Comment:
I think I have reviewed all the new katas you added and confirmed the levels for most of them. Please let me know if I missed something - my memory of what I reviewed a couple of days ago is quite bad.